test/statetest/statetest_loader.cpp

```diff
@@ -300,7 +297,8 @@ TestState from_json<TestState>(const json::json& j)
 /// Load common parts of Transaction or TestMultiTransaction.
 static void from_json_tx_common(const json::json& j, state::Transaction& o)
 {
-    o.sender = from_json<evmc::address>(j.at("sender"));
+    // Invalid block tests have only v, r, s, no sender.
```

Comment:
```suggestion
    // Sender is not resolved in transactions from invalid blocks.
```

Reply:
Are you attached to the particular word "resolved". I would rather see "included" or "present"?

Reply:
`recovered` is equally (or even more) confusing as resolved. We're looking at test JSON files here, and sometimes `sender` is just not provided. Sentence with "recovered" could wrongly imply that for tests with valid blocks it is recovered during JSON reading or sth.

Reply:
"provided" is fine to me.